src/Chapter2.hs

```diff
@@ -500,7 +504,9 @@ True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
+isThird42 :: [Int] -> Bool
+isThird42 (_ : _ : x : _ ) = x == 42
```

Comment:
You don't need to compare `x` with 42 separately, you can pattern match on `42` directly in the pattern.